homeassistant/components/webostv/__init__.py

```diff
@@ -138,15 +140,30 @@ async def async_on_stop(event):
         client.clear_state_update_callbacks()
         await client.disconnect()
 
-    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, async_on_stop)
+    async def async_load_platforms(_):
+        """Load platforms and event listener."""
+        await async_connect(client)
 
-    await async_connect(client)
-    hass.async_create_task(
-        hass.helpers.discovery.async_load_platform("media_player", DOMAIN, conf, config)
-    )
-    hass.async_create_task(
-        hass.helpers.discovery.async_load_platform("notify", DOMAIN, conf, config)
-    )
+        if client.connection is None:
+            async_call_later(hass, 60, async_load_platforms)
+            _LOGGER.warning(
```

Comment:
Should this be another level, if logged at all?

Reply:
Yes, definitely :P
I would say ideally warning only the firs time and debug after. While the fix appears to work, it's awfully spammy right now